pulsar-functions/worker/src/test/java/org/apache/pulsar/functions/worker/rest/api/v2/FunctionApiV2ResourceTest.java

```diff
@@ -282,22 +263,6 @@ public void testRegisterFunctionMissingClassName() throws IOException {
                 "Field 'className' cannot be null!");
     }
 
-    @Test
-    public void testRegisterFunctionMissingParallelism() throws IOException {
```

Comment:
why are we deleting these tests?

Reply:
Because some of the variables are implicit now(like parallelism). Thus their absence in the functionconfig is not an error

Reply:
ok